sample/src/main/java/com/alfresco/sample/AuthConfig.kt

```diff
@@ -6,8 +6,8 @@ val AuthConfig.Companion.defaultConfig: AuthConfig
     get() = AuthConfig(
         https = true,
         port = "443",
-        clientId = "zTSfC3VEcrguvP5CKIgNIBCOTliawIKg",
-        realm = "",
-        redirectUrl = "demo://dev-ps-alfresco.auth0.com/android/com.alfresco.sample/callback",
+        clientId = "alfresco-android-acs-app",
```

Comment:
These values should replace with placeholder, it should not contain demo server specific value